drupal/settings/production.settings.php

```diff
@@ -49,3 +49,8 @@
 if (is_numeric($file_gc = getenv('GOVCMS_FILE_TEMP_MAX_AGE'))) {
   $config['system.file']['temporary_maximum_age'] = $file_gc;
 }
+
+$config['lagoon_logs.settings']['host'] = 'application-logs.lagoon.svc';
```

Comment:
Can we use a environment variable with the default set to `application-logs.lagoon.svc`?